presto-parser/src/main/java/com/facebook/presto/sql/tree/FunctionCall.java

```diff
@@ -27,57 +27,55 @@
     private final QualifiedName name;
     private final Optional<Window> window;
     private final Optional<Expression> filter;
+    private final List<SortItem> sortItems;
```

Comment:
I think you should use `OrderBy` here, and that should be also `Optional` in the same manner as it is in `WindowNode`